### PR TITLE
Update join page

### LINKED ISSUE #1392: Update the Join page
### Overview
We need to update the Join page so that it will be ready to go live

### Action Items
- [x] In the "Advise Us" page card, remove step 3 and replace step 2 with the text below (under Instructions)
- [x] Remove the links from all of the role titles in the "Volunteer with Us" page card (they are linked to filter results that are not up-to-date)

### Resources/Instructions
[Join page](https://www.hackforla.org/join)

Updated "How to get involved" section should look like this now (new step 2 with # admin bolded, no step 3):

**EDIT**: Sorry I know this is inconsistent with the screenshot below, but can both mentions of slack actually be capitalized (Slack)? It seems like it's supposed to be capitalized based on googling around..

Screen Shot 2021-04-10 at 9 53 50 PM

Updated "Volunteer with Us" page card should look like this now (no links):

Screen Shot 2021-04-10 at 9 53 29 PM

### PR DESCRIPTION
**Pull request by danyc23**
fixes #1392 

Screen Shot 2021-04-14 at 12 18 20 PM

### CODE REVIEW

**Review by erikaBell — COMMENTED** [review]
@danyc23 looking good overall! I do have one question though, if I can pull @daniellex0 into it. I noticed one of the changes in this branch was a change on where a user is redirected when clicking on the "Project Volunteer" button 
<details>
<summary>Project Volunteer Btn</summary>

Screen Shot 2021-04-14 at 2 30 43 PM

Screen Shot 2021-04-14 at 2 32 34 PM
</details> 

On our live site, it redirects the user to "Getting Started"
<details>
<summary>Getting Started</summary>

Screen Shot 2021-04-14 at 2 30 53 PM
</details>

In Daniel's branch, it redirects the user to "/#projects"
<details>
<summary>Projects</summary>

Screen Shot 2021-04-14 at 2 31 00 PM
</details>

I did not see any mention of this change in the issue itself, or in any comments. I just want to clarify for myself if that change should be there?

If it was mentioned somewhere I didn't catch, feel free to direct me there. 

Besitos ~ ~

**Review by qiqicodes — CHANGES_REQUESTED** [review]
Changes looks good. 

On one note:
The href link is different from the current master branch, on line 63 is  "https://www.hackforla.org/#projects" 

On master branch is "https://www.hackforla.org/getting-started"

**Comment by danyc23** [comment]
I already made the requested changes. Let me know if there is anything else. Thanks!

**Review by danyc23 — COMMENTED** [review]
The changes from line 63 are done.

File: pages/join-us.html [hunk 1]
```diff
@@ -44,85 +36,57 @@ <h1>Advise Us</h1>
         </p>
         <p class='join-us-remove-p-padding'>How to get involved:</p>
         <ol>
-          <li>Join our <a href="https://hackforla-slack.herokuapp.com/" target="_blank">slack</a> (if you are comfortable using slack)</li>
+          <li>Join our <a href="https://hackforla-slack.herokuapp.com/" target="_blank">Slack</a> (if you are
```
Comment (daniellex0) [inline pages/join-us.html]: Please remove the parentheses section "(if you are comfortable using slack)" - sorry this is a recent change, Bonnie wants to remove this. So this line should just say "Join our Slack"

**Review by daniellex0 — CHANGES_REQUESTED** [review]
This looks great, just one additional change, sorry this is new- Please remove the parentheses section "(if you are comfortable using slack)" - Bonnie wants it removed. So this line should just say "Join our Slack." Other than that, looks good, thanks!

File: pages/join-us.html (hunk 1 above)
Comment (danyc23) [reply pages/join-us.html]: Hey, the changes are done. Thanks!!

**Review by akibrhast — APPROVED** [review]
LGTM

**Pull request merged**